### PR TITLE
Starting to make motivations explicit

### PR DESCRIPTION
**Pull request by gvwilson**
When done, this will address #44, but we need motivations for all the other sections.

### CODE REVIEW

**Comment by lexnederbragt** [comment]
Soms suggestions.

For Data Management:
1. Data should be [Findable, Accessible, Interoperable and Reusable](http://datafairport.org/fair-principles-living-document-menu)
2. Automating steps to munge data enhances transparency

For Software:
1. Help those you would like to reuse your code, and your future self, by making your code findable, documented and understandable
2. Modular code is easier to understand and troubleshoot and easier to make robust

For Collaboration (partly this text was already there):
1. Make it easy for other people to help you and give you credit for your work.
2. Help yourself as well as other people---remember,
   your most important collaborator is yourself six months from now.

For Project Organization
1. A well organised project makes elements easy to find.

For Version Control, I had written (and moved from existing text):
1. Version control allows you to store snapshots of the states of your project, meaning you can check its history and turn back the clock if you need to
2. Version control more thoroughly documents changes, when they happened and who made them than Track Changes (Word, Google Docs) and document history (Dropbox, Google Docs and others)
3. It is useful even when you're working alone because it lets your past self collaborate with you.
   - For example, comparing ("diff'ing") the submitted and current revisions of a paper help with preparing a list of changes for the reply to referees.

For Manuscripts:
1. A single 'master' document that everyone can work/comment on, works better than sending different versions around to collect comments
2. Avoid the possibility of overwriting each others contributions

**Comment by gvwilson** [comment]
@lexnederbragt can you please have a look?

**Comment by pdebuyl** [comment]
I find that too often version control is presented for its history logging features and not for checking, simply, the _current state_ of the code. So, to

```
version control of code, figures, and data helps you figure out why Figure 4 looks
different now from last week.
```

I would add

```
In addition to acting as a powerful undo function, version control lets you verify that
you are running a known version of your code, that is the latest registered version of
your code.
```

File: index.md [hunk 1]
```diff
@@ -136,8 +165,25 @@ Regarding metadata, [Elizabeth Wickes][wickes-comment-metadata] writes:
...
+### Goals
+
+1.  Again, the easier it is for people to collaborate, the more likely they are to do so
+    (and to give you credit)
+2.  Two most reported barriers from [Steinmacher et al][steinmacher-newcomers]
+    are finding a task to start on
+    and problems setting up the local workspace to start work
+    *   So remove those
+3.  Additional disincentive is uncertainty: what am I allowed to do?
+    *   So remove that as well
```
Comment (lexnederbragt) [inline index.md:185]: Maybe add that having no license basically says "I am keeping all copyright and you are not allowed to do anything"?

**Comment by lexnederbragt** [comment]
Much more eloquent than I phased it :-). Looks really good.

The current phrasing of "at the minimum every researcher _should_ do" (emphasis added) and the us of goals and rules sections, could this potentially scare off the audience? Are we aiming to tell researchers what to do, or to show them what can be done to make their work more productive, reusable, reproducible etc? How about a "set of guidelines to get you started" with subsections "Motivation", "Practices" and "Discussion"?

**Comment by lexnederbragt** [comment]
@pdebuyl I am not sure, but maybe you refer to the case where someone is on a different branch than master? Branching is explicitly not included. Without it, how can someone not use the "the latest registered version" of the code?

**Comment by pdebuyl** [comment]
@lexnederbragt I meant it in the sense of clean/unclean state of the working directory. Even with only a single branch, it is good to know that the files you are compiling/executing are the ones you think they are.

**Comment by lexnederbragt** [comment]
@pdebuyl Of course, I hadn't thought of that.

**Pull request merged**